### DISCUSSION TITLE
Iden of pins conecting lora and esp32 without source code

### DISCUSSION THREAD

**Discussion by anujowa** [jgromes/RadioLib#619] [Q&A]
I have the bin file for a code loaded on an esp32. The esp32 was connected to a nicerf sx1268 lora chip. I can't figure out which pin of lora and esp32 were connected. How can I find out thr same without having the original source code with me

**Comment by jgromes** [comment]
I don't see any realistic way how to do that. If you have the original .elf file that was produced during compilation, then maybe some dissasembly to identify digital IO register accesses could be attempted, but I don't think it's worth the trouble.

**Comment by anujowa** [comment]
Thank you for your reply. I was wondering if i can match thebpins by
comparing the signals using a DSO. Managed to figure out pins for sck and
mosi. They were simillar to vspi. So connected the remaining pins as per
the standard. But problem is iden of where to connect busy pin as it is not
part of standard spi configuration.
